src/parse.js

```diff
@@ -1,16 +1,17 @@
 const InvalidInstructionException = require('./commands/invalidInstructionException');
 const parse = instruction => {
   let empty = /^\s*$/;
+  let onlyNumber = /^\s*([0-9]+)+\s*$/;
   let comment = /^\s*;.*$/;
   let components = /^\s*([0-9]+)\s+([a-zA-Z]+)\s*(.*)*$/;
-  if (instruction.match(empty) || instruction.match(comment))
-    return { nonExecutableLine: true };
+  if (instruction.match(empty) || instruction.match(comment) || instruction.match(onlyNumber))
```

Comment:
It would be better to refactor this line as:

```javascript
if (isNonExecutable(instruction))
  return { nonExecutableLine: true }
```

and extract `isNonExecutableLine` as its own function